agent/battle/execution/coords.py

```
@@ -0,0 +1,130 @@
+# 换人界面（Order Change）选择从者点击点 (1..6)
+ORDER_CHANGE_MEMBER = {
+    slot: center(roi) for slot, roi in ORDER_CHANGE_MEMBER_ROI.items()
+}
+ORDER_CHANGE_CONFIRM_ROI = (559, 592, 151, 58)
+ORDER_CHANGE_CONFIRM_BTN = center(ORDER_CHANGE_CONFIRM_ROI)
+ORDER_CHANGE_CANCEL_BTN = center((1203, 123, 30, 24))
+
+# ---- 战斗结算流程（胜利后：掉落/羁绊/结果多屏点击直到回关卡列表）----
+# 标定护栏：未真机标定前保持 False；runtime 在未标定时不会盲点，仍按"识别到胜利即返回成功"处理。
+SETTLEMENT_CALIBRATED = False
+# TODO(标定): 结算各屏"继续/下一步"的安全点击点（1280x720）。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)